crates/bevy_render/src/camera/camera.rs

```diff
@@ -75,14 +75,49 @@ pub struct ComputedCameraValues {
     old_viewport_size: Option<UVec2>,
 }
 
+/// How much energy a [`Camera3d`] absorbs from incoming light.
+///
+/// <https://en.wikipedia.org/wiki/Exposure_(photography)>
 #[derive(Component)]
 pub struct ExposureSettings {
+    /// <https://en.wikipedia.org/wiki/Exposure_value#Tabulated_exposure_values>
+    pub ev100: f32,
+}
+
+impl ExposureSettings {
```

Comment:
Could you add a few associated constants? Say:

```rust
pub const EV100_SUNLIGHT: f32 = 15.0;
pub const EV100_OVERCAST: f32 = 12.0;
pub const EV100_INDOOR: f32 = 7.0;
```
And then define the default in terms of `EV100_OVERCAST` rather than camera parameters? (The current default is very close to 12.0 but figuring that out is a bit annoying)

Reply:
I'm not sure how I feel about associated constants tbh. We don't do that for directional or point lights, for instance.

Reply:
Perhaps we should? :) We have a whole bunch of colours for example. Basically curated presets from which people can select, but people can also set whatever they like.

Reply:
I'll add the EV100 associated constants then.